Kube proxy (#110)
OpenLiberty/draft-guide-kubernetes-microprofile-health: README.adoc

// Static guide instruction
ifndef::cloud-hosted[]

> An endpoint has been provided under the `system` microservice to set it to an unhealthy 

Reviewer: ```suggestion
An `unhealthy` endpoint has been provided under the `system` microservice to set it to an unhealthy 
```

> state in the health check. The unhealthy state will cause the readiness probe to fail.

Reviewer: ```suggestion
state. The unhealthy state will cause the readiness probe to fail.
A request to the `unhealthy` endpoint will put the service in an unhealthy state as a simulation.
```

> The endpoint should be implemented by a POST request.

Reviewer: ```suggestion
```

> For the reason of simplicity, the endpoint is implemented by a GET request. 

Reviewer: ```suggestion
```

> 

Reviewer: I suggest that we provide only 1 curl command regardless of platform to make it simpler.
